apps/crn-frontend/src/events/__tests__/Events.test.tsx

```diff
@@ -91,25 +68,6 @@ describe('Events', () => {
     );
   });
 
-  it('Defaults to the upcoming events page', async () => {
```

Comment:
Why was this test removed. It doesn't seem like a legacy test at first glance

Reply:
It is legacy, the `getEvents` was used in `apps/crn-frontend/src/events/state.ts` when the flag was disabled.